packages/ckeditor5-list/tests/utils.js

```diff
@@ -133,4 +137,313 @@ describe( 'utils', () => {
 			} );
 		} );
 	} );
+
+	describe( 'getSiblingListItem()', () => {
```

Comment:
Well, we had integration tests for that so it was covered.

While not super crucial I'm OK with that lately. As long as top-level unit tests all the branches of this it might be OK to not have direct tests. But that's probably on a case-by-case basis.

Having "unit" defined as a whole ListStyle feature it is easier to refactor internals (including util methods) without changing tests (the definition of the feature).
OTOH, having those tests at util function level helps assuring that everything works OK.